templates/includes/useful-info.html

```diff
@@ -0,0 +1,22 @@
+{% load wagtailimages_tags wagtailcore_tags %}
+
+<div class="tna-container">
+    <div class="tna-useful-info {% if image %}tna-useful-info--with-image{% endif %}">
+        <div class="tna-useful-info__content">
+            <h2 class="tna-heading-l">Need to know</h2>
+    
+            <h3 class="tna-heading-s">Location</h3>
+            <p>There is parking available on-site, but spaces are limited, so visitors are encouraged to use public transport where possible.</p>
+    
+            <h3 class="tna-heading-s">Accessibility</h3>
+            <p>We are committed to making our tours accessible to all visitors, with provisions in place for people with disabilities or mobility issues. If you have any specific accessibility requirements, please get in touch with The National Archives in advance to discuss your needs.</p>
+    
+            <a href="/" class="tna-useful-info__cta tna-button tna-button--dark">Planning your visit</a>
+        </div>
+        {% if image %}
```

Comment:
I think because of the 'if' expression here I'm not seeing the hard-coded image when I preview the page. I think the back-end data is in place so we could hook that in now rather than using hard-coded data.

Reply:
I've realised the back-end might need either some extra fields, or to make this rich text.